sonar-plugin-api/src/main/java/org/sonar/api/web/WidgetLayoutType.java

```diff
@@ -21,7 +21,9 @@
 
 /**
  * @since 2.4
+ * @deprecated since 6.2, as dashboards have been removed
  */
+@Deprecated
 public enum WidgetLayoutType {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)